crates/trigger-http/src/lib.rs

```diff
@@ -583,6 +586,8 @@ struct ChainedRequestHandler {
 pub struct HttpRuntimeData {
     origin: Option<String>,
     chained_handler: Option<ChainedRequestHandler>,
+    /// If provided, these options used for client cert auth
```

Comment:
nit: can you say in the comment what the `String` in the hashmap is for:

```suggestion
    /// Optional mapping of authority and TLS options 
```

Reply:
added a comment `Optional mapping of authority and TLS options for the current component`